prophy/kind.py

```diff
@@ -0,0 +1,16 @@
+class Constant(type):
```

Comment:
Kind needn't have attribute setting blocked, it's a very unlikely error to happen.